Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -750,60 +695,6 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
       ) ?? Default.enumCases
     }
   }
-
-  public struct OperationDocumentFormat: OptionSet, Codable, Equatable {
```

Comment:
I don't think `OperationDocumentFormat` should be moved to the manifest config. This is something that actually affects the way _code_ is generated and is not actually even used when generating the operation manifest itself.

Reply:
My thinking was there was no use case for this outside of persisted queries, although that may be an incorrect assumption? Technically I guess you could do APQs with no manifest and still need this document for option yea?